app/config/config.yml

```diff
@@ -16,7 +16,7 @@ broadway:
         repository: elasticsearch
         elasticsearch:
             hosts:
-                - elasticsearch:9200
```

Comment:
it needs to be `localhost` when using Vagrant and `elasticsearch` when using docker?